src/app/lib/config/services/afaanoromoo.js

```diff
@@ -184,6 +184,9 @@ export const service = {
       hasRadioSchedule: true,
       onFrontPage: false,
     },
+    wsoj: {
```

Comment:
We might want to name this `recommendations` instead of the old wsoj terminology